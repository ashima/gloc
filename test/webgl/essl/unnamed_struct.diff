--- unnamed_struct.true	2011-10-30 19:15:30.000000000 -0700
+++ unnamed_struct.out	2011-11-17 16:38:57.000000000 -0800
@@ -1 +1,9 @@
-error
+precision highp float;
+
+struct { int q; };
+
+uniform struct { float k; } r;
+
+struct { float k; } id(struct { float k; } k) { return k; }
+
+void main() { id(r); }
