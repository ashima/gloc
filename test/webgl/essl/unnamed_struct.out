#line 1 0/* <stdin>#u=0 */
precision highp float;

struct { int q; };

uniform struct { float k; } r;

struct { float k; } id(struct { float k; } k) { return k; }

void main() { id(r); }

